sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/keyvault/environment/KeyVaultOperation.java

```diff
@@ -119,7 +119,7 @@ public String[] getPropertyNames() {
             return properties
                 .keySet()
                 .stream()
-                .flatMap(p -> Stream.of(p, p.replaceAll("-", ".")))
```

Comment:
refer: https://sonarcloud.io/project/issues?directories=sdk%2Fspring%2Fspring-cloud-azure-autoconfigure%2Fsrc%2Fmain%2Fjava%2Fcom%2Fazure%2Fspring%2Fcloud%2Fautoconfigure%2Fkeyvault%2Fenvironment&resolved=false&types=CODE_SMELL&id=stliu_azure-sdk-for-java&open=AYAHSa95i07DX7oNprfr